test/generic_test/nvs.csv

```diff
@@ -1,3 +1,2 @@
 key,type,encoding,value
 twomes_storage,namespace,,
-mac_addresses,data,string,ab:cd:ef:01:23:45
```

Comment:
@henriterhofte is it not useful to leave an example of how to use this file? Maybe even add a second one to show how to use the `;` to separate them.